xmtp_mls/Cargo.toml

```diff
@@ -32,3 +32,4 @@ anyhow = "1.0.71"
 
 [dev-dependencies]
 tempfile = "3.5.0"
+xmtp_api_grpc = { path = "../xmtp_api_grpc" }
```

Comment:
I'm proposing for this MLS work we ditch the `MockApiClient` and just use a local server for all testing.

Reply:
It means we always have to run a server in order to run tests, which is fine by me. The reason is because the server logic is a little more complex and it would be duplicative to mock out?

Reply:
That's basically it. The mock API client was already incomplete (didn't really support pagination). Would be even harder to properly mock out the new APIs. This way, we get real validation of bad messages etc.